templates/arm-mon-custom-alerts.json

```diff
@@ -270,6 +272,90 @@
                     }
                 }
             }
+        },
+        {
+            "name": "[concat(parameters('aksName'), ' - ', 'Pods in not Running state (Pending, Failed, Unknown)')]",
+            "type": "Microsoft.Insights/scheduledQueryRules",
+            "apiVersion": "2018-04-16",
+            "location": "[resourceGroup().location]",
+            "tags" : {
+                "EnvironmentName": "[parameters('environmentName')]",
+                "Branch":"[parameters('branch')]",
+                "managedBy":"[parameters('managedBy')]",
+                "solutionOwner": "[parameters('solutionOwner')]",
+                "activityName": "[parameters('activityName')]",
+                "dataClassification":"[parameters('dataClassification')]",
+                "automation": "[parameters('automation')]",
+                "costCentre":"[parameters('costCentre')]",
+                "environment": "[parameters('environment')]",
+                "criticality": "[parameters('criticality')]"
+             },
+            "properties": {
+                "description": "Pods that are failing for at least 1 hour with not running state.",
```

Comment:
don't we want to know sooner than an hour?

Reply:
I have reduced the timeWindows to 15 minutes and the frequency to 5 minutes ... We can change them later to whichever value we think is appropriate. Thanks